docs/Data_Portal/Users_Guide/Developers_Guide.md

```diff
@@ -854,461 +1007,475 @@ the state uses [Redux Toolkit] and [Redux Persist] to store the state in local s
+                          {pickedProjects.length}{" "}
+                      </CountsIcon>
+                  ) : null
+              }
+              onClick={() => setShowSaveCohort(true)}
+              className="border-primary data-disabled:opacity-50 data-disabled:bg-base-max data-disabled:text-primary"
           >
             Save New Cohort
           </Button>
         </span>
             </Tooltip>
             {showSaveCohort && (
-                    <SaveCohortModal
-                            onClose={() => setShowSaveCohort(false)}
-                            filters={{
-                              mode: "and",
-                              root: {
-                                "cases.project.project_id": {
-                                  operator: "includes",
-                                  field: "cases.project.project_id",
-                                  operands: pickedProjects,
-                                },
-                              },
-                            }}
-                    />
+                <SaveCohortModal
+                    onClose={() => setShowSaveCohort(false)}
+                    filters={{
+                        mode: "and",
+                        root: {
+                            "cases.project.project_id": {
+                                operator: "includes",
+                                field: "cases.project.project_id",
+                                operands: pickedProjects,
+                            },
+                        },
+                    }}
+                />
             )}
-          </>
-  );
+        </>
+    );
 };
 
 export default ProjectsCohortButton;
 ```
 
-This custom button component used the state variable `showSaveCohort` to determine if the `SaveCohortModal` component needs to be shown.
-The `SaveCohortModal` component is passed the current list of projects selected by the user and handles the creation of the cohort and saving it.
+This custom button component uses the state variable `showSaveCohort` to determine if the `SaveCohortModal` component
+needs to be shown.
+The `SaveCohortModal` component is passed to the current list of projects selected by the user and handles the creation of
+the cohort and saving it.
 
 ### Application Demo
 
-In addition to a application that works on cohorts, an application can have a demo. This demo can be used to show the
-application's functionality. The demo is shown when the demo button is clicked. The demo button is shown when the
-application is registered with `hasDemo: false,` as described in the [Application Registration](#application-registration) section.
+In addition to the actual application, it can have a demo. The demo can be used to show the
+application's functionality and is shown when the demo button is clicked. The demo button is shown when the
+application is registered with `hasDemo: false,` as described in
```

Comment:
`hasDemo: true`?

Reply:
Good catch. Not sure why that was ever written as false.